src/dmd/frontend.h

```diff
@@ -6085,14 +6035,8 @@ class PostorderStatementVisitor final : public StoppableVisitor
     void visit(LabelStatement* s);
 };
 
-enum : bool { LOG = false };
-
-enum : bool { LOG = false };
```

Comment:
This would be another example of why `extern(D)` enum constants should be ignored.